gems/manageiq-providers-amazon/spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb

```diff
@@ -41,35 +42,35 @@
   end
 
   def assert_table_counts
-    expect(ExtManagementSystem.count).to eq(1)
+    expect(ExtManagementSystem.count).to eq(2)
     expect(Flavor.count).to eq(55)
     expect(AvailabilityZone.count).to eq(5)
     expect(FloatingIp.count).to eq(5)
-    expect(AuthPrivateKey.count).to eq(11)
-    expect(CloudNetwork.count).to eq(4)
-    expect(CloudSubnet.count).to eq(6)
+    expect(AuthPrivateKey.count).to eq(12)
+    expect(CloudNetwork.count).to eq(5)
+    expect(CloudSubnet.count).to eq(10)
     expect(OrchestrationTemplate.count).to eq(2)
     expect(OrchestrationStack.count).to eq(2)
     expect(OrchestrationStackParameter.count).to eq(5)
     expect(OrchestrationStackOutput.count).to eq(1)
     expect(OrchestrationStackResource.count).to eq(24)
-    expect(SecurityGroup.count).to eq(33)
-    expect(FirewallRule.count).to eq(88)
-    expect(VmOrTemplate.count).to eq(46)
-    expect(Vm.count).to eq(26)
+    expect(SecurityGroup.count).to eq(36)
+    expect(FirewallRule.count).to eq(94)
+    expect(VmOrTemplate.count).to eq(47)
+    expect(Vm.count).to eq(27)
     expect(MiqTemplate.count).to eq(20)
 
     expect(CustomAttribute.count).to eq(0)
     expect(Disk.count).to eq(14)
     expect(GuestDevice.count).to eq(0)
-    expect(Hardware.count).to eq(46)
-    expect(Network.count).to eq(13)
+    expect(Hardware.count).to eq(47)
+    expect(Network.count).to eq(15)
     expect(OperatingSystem.count).to eq(0) # TODO: Should this be 13 (set on all vms)?
     expect(Snapshot.count).to eq(0)
     expect(SystemService.count).to eq(0)
 
-    expect(Relationship.count).to eq(25)
-    expect(MiqQueue.count).to eq(48)
+    expect(Relationship.count).to eq(26)
+    expect(MiqQueue.count).to eq(50)
```

Comment:
yeah :-) I think I am using the same env as you

Btw. in openstack infra I changed the craziest checks just for > 0. For the rest it should be enough we are checking the specific entities we have in the env.